### PR TITLE
SNOW-759237 edit few log lines

### PR DESCRIPTION
**Pull request by sfc-gh-gdoci**
This PR removes "Arrow" keyword in non-arrow specific logging and replaces it with Parquet in one place.

### CODE REVIEW

File: src/main/resources/net/snowflake/ingest/ingest_error_messages.properties [hunk 1]
```diff
@@ -6,7 +6,7 @@
 0001=Ingest client internal error: {0}.
 0002=Required value is null, Key: {0}.
 0003=Required value is empty, Key: {0}.
-0004=The given row cannot be converted to Arrow format: {0}. {1}
+0004=The given row cannot be converted to Parquet format: {0}. {1}
```
[the remark this answers is not in the document]
Comment (sfc-gh-tzhang) [reply src/main/resources/net/snowflake/ingest/ingest_error_messages.properties:9]: Yeah, we still use this for Arrow currently, right?

**Review by sfc-gh-tzhang — COMMENTED** [review]
Thanks @sfc-gh-gdoci , I think we also have some comments that need to be updated if you search for Arrow, for example: `We need to update the start offset for the EP and Arrow data ` or  `for (byte[] arrowData : chunksDataList)`

File: src/main/resources/net/snowflake/ingest/ingest_error_messages.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (sfc-gh-gdoci) [reply src/main/resources/net/snowflake/ingest/ingest_error_messages.properties]: yes, changed it to `0004=The given row cannot be converted to the internal format: {0}. {1}`.

**Review by sfc-gh-tzhang — APPROVED** [review]
LGTM, thanks!

**Pull request merged**